packages/db-objects-redis/lib/objects/objectsInRedisClient.js

```diff
@@ -1358,7 +1370,13 @@ class ObjectsInRedisClient {
             // it is file
             const metaID = this.getFileId(id, name, true);
             const dataID = this.getFileId(id, name, false);
-            this.delObject(dataID, _err => this.delObject(metaID, callback));
+            try {
+                await this.delObject(dataID);
+                await this.delObject(metaID);
```

Comment:
Formally different as before, but ok in this case I think ... if data can not be deleted we should also not delete the meta part . can now only happyn other way around

Reply:
hm i can also make promise all then it is like before

Reply:
will be another pr